main/src/cgeo/geocaching/command/RemoveNotUniqueCommand.java

```diff
@@ -0,0 +1,48 @@
+package cgeo.geocaching.command;
+
+import android.app.Activity;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.Set;
+
+import cgeo.geocaching.R;
+import cgeo.geocaching.SearchResult;
+import cgeo.geocaching.enumerations.LoadFlags;
+import cgeo.geocaching.models.Geocache;
+import cgeo.geocaching.storage.DataStore;
```

Comment:
Please sort the imports as stated in https://github.com/cgeo/cgeo/wiki/Coding-Conventions